### PR TITLE
Change the way to create groovy editor buttons

### PR DESCRIPTION
**Pull request by adrbessy**
Button tooltips are visible, 
ClassPath buttons are separated from the others

### CODE REVIEW

File: plugins/org.orbisgis.h2gis/plugin.xml [hunk 1]
```diff
@@ -66,18 +66,18 @@
...
+					<file type="jar" path="maven:/org.orbisgis:h2gis-osgi:2.0.0"/>
+                    <file type="jar" path="maven:/org.orbisgis:h2gis-utilities:2.0.0"/>
+                    <file type="jar" path="maven:/org.orbisgis:postgis-jts:2.0.0"/>
+					<file type="jar" path="maven:/org.orbisgis:postgis-jts-osgi:2.0.0"/>
+					<file type="jar" path="maven:/org.orbisgis:cts:1.6.0"/>
+					<file type="jar" path="maven:/org.orbisgis:poly2tri:0.4.0"/>
+					<file type="jar" path="maven:/org.orbisgis:java-network-analyzer:0.3.0"/>
                     <file type="jar" path="maven:/org.slf4j:slf4j-simple:1.7.32"/>
-                    <file type="jar" path="maven:/org.slf4j:slf4j-api:1.7.32"/>
+                    <file type="jar" path="maven:/org.slf4j:slf4j-api:1.7.35"/>
```
Comment (ebocher) [inline plugins/org.orbisgis.h2gis/plugin.xml:80]: H2GIS uses 1.7.32

File: plugins/org.orbisgis.h2gis/bin/plugin.xml [hunk 2]
```diff
@@ -66,23 +66,23 @@
...
                     <file type="jar" path="maven:/org.locationtech.jts:jts-core:1.18.2"/>
-                    <file type="jar" path="maven:/org.jgrapht:jgrapht-core:0.9.2"/>
+                    <file type="jar" path="maven:/org.jgrapht:jgrapht-core:1.0.1"/>
                     <file type="jar" path="maven:/com.fasterxml.jackson.core:jackson-core:2.13.0"/>
                     <file type="jar" path="maven:/org.apache.commons:commons-compress:1.21"/>
-                    <file type="jar" path="maven:/com.h2database:h2:2.0.206"/>
+                    <file type="jar" path="maven:/com.h2database:h2:2.1.210"/>
 
                     <parameter name="query-get-active-db" value="SELECT SCHEMA()"/>
                     <parameter name="query-set-active-db" value="SET SCHEMA = ?"/>
```
Comment (ebocher) [inline plugins/org.orbisgis.h2gis/bin/plugin.xml:88]: This file must be in gitignore

File: plugins/org.orbisgis.groovyeditor/pom.xml [hunk 3]
```diff
@@ -31,6 +30,11 @@
 			<artifactId>org.orbisgis.core</artifactId>
 			<version>1.0.0-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+		    <groupId>org.eclipse.lsp4j</groupId>
```
Comment (ebocher) [inline plugins/org.orbisgis.groovyeditor/pom.xml:34]: Not necessary in this PR ?

File: plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/GroovyJob.java [hunk 4]
```diff
@@ -157,28 +121,19 @@ public void run() {
...
-            } catch(Exception e){
+            } catch (Exception e){
                 e.printStackTrace(pw);
                 String sStackTrace = sw.toString();
+            	GroovyConsoleContent.writeFileNameIntoConsole(name);
                 GroovyConsoleContent.writeIntoConsole(sStackTrace);
-                LocalDateTime now = LocalDateTime.now();
-                outputPrintWriter.write("\n" + dtf.format(now) + "\n" + sStackTrace);
-                GroovyConsoleContent.writeIntoConsole("BAD_END");   
+                GroovyConsoleContent.writeIntoConsole("BAD_END");
```
Comment (ebocher) [inline plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/GroovyJob.java:133]: It seems that this PR is not up to date with the master
https://github.com/orbisgis/orbisrcp/blob/master/plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/GroovyJob.java#L136

**Comment by ebocher** [comment]
Some conflicts must be fixed to enable the merge

**Review by SPalominos — APPROVED** [review]

**Pull request merged**